content/modals/AU/short_term.json

```
@@ -0,0 +1,38 @@
+{
+    "meta": {
+        "product": "PAY_LATER_SHORT_TERM",
+        "periodicPayment": "{formattedPeriodicPayment}",
+        "minAmount": "{minAmount}",
+        "maxAmount": "{maxAmount}",
+        "qualifying": "{qualifying_offer}",
+        "amount": "{transaction_amount}",
+        "apr": "{apr}",
+        "variables": {
+            "transaction_amount": "${eval(transaction_amount ? transaction_amount : '-')}",
+            "qualifying_offer": "${eval(CREDIT_OFFERS_DS.qualifying_offer ? CREDIT_OFFERS_DS.qualifying_offer : 'false')}",
+            "financing_code": "${CREDIT_OFFERS_DS.financing_code}",
+            "formattedPeriodicPayment": "${CREDIT_OFFERS_DS.formattedPeriodicPayment}",
+            "total_payments": "${CREDIT_OFFERS_DS.total_payments}",
+            "formattedMinAmount": "${CREDIT_OFFERS_DS.formattedMinAmount}",
+            "formattedMaxAmount": "${CREDIT_OFFERS_DS.formattedMaxAmount}",
+            "formattedTotalCost": "${CREDIT_OFFERS_DS.formattedTotalCost}",
+            "minAmount": "${CREDIT_OFFERS_DS.minAmount}",
+            "maxAmount": "${CREDIT_OFFERS_DS.maxAmount}",
+            "apr": "${CREDIT_OFFERS_DS.apr}",
+            "nominal_rate": "${CREDIT_OFFERS_DS.nominal_rate}"
+        }
+    },
+    "content": {
+        "headline": "Pay in 4 interest-free payments",
+        "subheadline": "Available for purchases of {formattedMinAmount} to {formattedMaxAmount}. No sign-up fees or late fees.",
+        "qualifyingSubheadline": "Split your purchase of {formattedTotalCost} into {total_payments} with no sign-up fees or no late fees.",
+        "donutTimestamps": ["Today", "2 weeks", "4 weeks", "6 weeks"],
+        "instructions": [
+            "Choose PayPal at checkout to pay later with <strong>Pay in 4</strong>. ",
+            "Complete your purchase with a 25% first payment.",
+            "Remaining payments are taken automatically. It’s easy!"
+        ],
+        "disclosure": "PayPal Pay in 4 is available for eligible purchases of $30 to $1,500 AUD. Some seller exclusions apply and subject to customer eligibility. We may run a credit check when you apply. We don't charge late fees, but late or missed payments could impact your PayPal account and credit score. You must live in Australia, have an Australian PayPal account, and agree to all relevant terms and conditions to use Pay in 4.",
```

```suggestion
        "disclosure": "PayPal Pay in 4 is available for eligible purchases of {formattedMinAmount} to {formattedMaxAmount} AUD. Some seller exclusions apply and subject to customer eligibility. We may run a credit check when you apply. We don't charge late fees, but late or missed payments could impact your PayPal account and credit score. You must live in Australia, have an Australian PayPal account, and agree to all relevant terms and conditions to use Pay in 4.",
```

Let's pull in the formatted min and max into the disclosure.